addons/udes_stock/tests/plot_load_tests.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from scipy.stats import spearmanr
+from collections import defaultdict
+from matplotlib import pyplot as plt
+import sys
+
+COEFFICIENT_THRESHOLD = 0.7
+PVAL_THRESHOLD = 0.001
+
+# Do we want to bootstrap this?
+def scaling_coefficient(x, y):
+    """Nuno's neat method"""
+    dydx = np.diff(y)/np.diff(x)
+    coefficient, pval = spearmanr(x[1:], dydx)
+    area = np.sum(dydx - dydx[0])
+    return area, coefficient, pval
+
+
+def parse_time_file(filename):
+    lines = open(filename, 'r').read()
+    data = defaultdict(lambda: defaultdict(list))
+
+    for line in filter(lambda l: len(l) > 0 and '=' not in l,
```

Comment:
Given that a non-empty string will be True anyway, can this be just "lambda l: l and '=' not in l"?